### PR TITLE
DEV-26342: add checkingFrequency in all user model responses

### PR DESCRIPTION
**Pull request by george-haddad**
- Add `UserStatus` enum in data structure
- Add `checkingFrequency` attribute in all user model responses

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -4566,14 +4575,20 @@ Debug API:
...
++ checkingFrequency (UserStatus) - Displays a rough idea of the usage pattern of the user by checking their frequency across the lifetime of the user in days
 + properties (object) - Key/Value pair of properties
 + roles (array[Role]) - List of assigned roles
 + customFields (array[CustomField]) - List of custom fields
 
+### UserStatus (enum)
++ `frequent`
++ `infrequent`
++ `regular`
+
```
Comment (george-haddad) [inline apiary.apib:4591]: @tkrisztian95 check it out, enums :)

Comment (tkrisztian95) [reply apiary.apib:4591]: Cool! I like it :)

Comment (tkrisztian95) [reply apiary.apib:4591]: Can you please also add descriptions for these values? 
```suggestion
### UserStatus (enum)
+ `frequent` - Perform 1 more check than regular users.
+ `infrequent` - Perform fewer checks than regular users.
+ `regular` - Performs 1 check in 24 hours (plus/minus 25%) on average.

```
To get a rough idea about what are the enum values represent in regard document checks.

**Review by tkrisztian95 — COMMENTED** [review]
Looks good!

**Pull request merged**